setup.py

```diff
@@ -19,6 +19,7 @@
 except ImportError:
     cythonize = None
 
+sys.path += [os.path.dirname(__file__)]
```

Comment:
so I assume it didn't work without that even if you invoked stuff from the toplevel dir?

usually one should rather try not having to modify sys.path.

Reply:
Neither the CWD nor the parent of setup.py are on sys.path if we use setuptools.build_meta. This is, apparently, a feature. Either we append `os.path.dirname(__file__)` or we use `setuptools.build_meta:__legacy__`, which will do the same for us.

https://github.com/pypa/setuptools/discussions/3134#discussioncomment-2249652

> The default backend (setuptools.build_meta) will not add the project folder to the path on purpose.
> That is only supported in `setuptools.build_meta:__legacy__`

Reply:
ok, so keep it with line 22. hopefully this won't shoot ourselves in the knee at some time.